includes/Data/Data.php

```diff
@@ -105,4 +106,17 @@ public static function customer_data() {
 		}
 		 return array();
 	}
+
+	/**
+	 * Get the current theme data like the theme variations and previews per step
+	 *
+	 * @return array
+	 */
+	public static function theme_data()
+	{
+		$theme_step_data = Patterns::get_count_of_patterns();
+		$theme_step_data["theme-styles"]["styles"] = count(\WP_Theme_JSON_Resolver::get_style_variations()) + 1;
+		return $theme_step_data;
```

Comment:
Let's return an array similar to this `array( 'step_slug ' => array('previewCount' => combine necessary variables here))` instead of doing computation in the front end

Reply:
{
    "theme-styles": {
        "styles": 6,
        "patterns": 1
    },
    "homepage-styles": {
        "styles": 1,
        "patterns": 3
    },
    "site-pages": {
        "styles": 1,
        "patterns": 4
    }
}

This is how it is rn, do u wnt me to combine this here, cause I have done that in frontend as of now.